config/base/numaflow-server/numaflow-server-rbac-config.yaml

```diff
@@ -9,6 +9,13 @@ data:
     p, role:readonly, *, *, GET
     # Groups go here
     # g, admin, role:admin
+    g, sidhant.kohli@gmail.com, role:admin
     # g, my-github-org:my-github-team, role:readonly
   rbac.conf: |
     policy.default: role:readonly
+    // The scopes field controls which authentication scopes to examine during rbac enforcement.
```

Comment:
Does `#` work for the comments? Having both `#` and `//` in the same config map for comments is very confusing.